source/whats-new.txt

```diff
@@ -26,6 +27,49 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+New features of the 4.11 {+driver-short+} release include:
+
+- Recursive Schema Support
+
+  - The driver now supports recursive schemas and provides type safety
+    on dot notation queries up to a depth of 9. Beyond a depth of 9,
+    your code still compiles but is no longer type checked and falls
+    back to a type of ``any``. This depth limit is a product of
+    Typescript's recursive type limitations.
+
+    - Create an interface with a recursive schema:
+
+      .. code-block:: js
+
```

Comment:
Suggestion: use "copyable: false" property for all the code snippets in this section since they aren't attempting to demonstrate something that the user should try using copy and paste.